lib/cartopy/examples/gridliner.py

```diff
@@ -0,0 +1,40 @@
+"""
+Gridlines and tick labels
+-------------------------
+
+These examples demonstrate how to quickly add longitude
+and latitude gridlines and tick labels on a non-rectangular projection.
+
+As you can see on the first example,
+longitude labels may be drawn on left and right sides,
+and latitude labels may be drawn on bottom and top sides.
+Thanks to the ``dms`` keyword, minutes are used when appropriate
+to display fractions of degree.
+
+
+In the second example, labels are still drawn at the map edges
+despite its complexity, and some others are also drawn within the map
+boundary.
+
+"""
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.pyplot as plt
+
+
+def main():
+
+    rotated_crs = ccrs.RotatedPole(pole_longitude=120.0, pole_latitude=70.0)
+    ax0 = plt.axes(projection=rotated_crs)
+    ax0.set_extent([-3, 2, 48, 52], crs=ccrs.PlateCarree())
```

Comment:
This appears to be the middle of nowhere (as far as any details in the land shapefiles go); can you change it to somewhere that doesn't end up as a completely land-filled image?

Also, maybe set the land to use `100m` resolution, because otherwise it downloads 10m which is rather large.

Reply:
It was centered on the Channel. 
It is now moved sligthly westward with a greater extent that gives continents a not too crude shape with the 1:110m resolution.